src/extensions/component-enumerators/examples/contoso-component-enumerator/demo/sample-updates/CreateSampleMSOEUpdate-8.x.ps1

```diff
@@ -150,7 +150,7 @@ Write-Host "Preparing update $parentUpdateIdStr ..."
     # ADD STEP(S)
     #
 
-    # This update contains 3 steps.
+    # This update contains 2 steps.
```

Comment:
maybe remove comment to avoid need to update when add/rmv step later?

Reply:
Removed